src/install-peerdeps.js

```diff
@@ -154,12 +154,9 @@ const getPackageString = ({ name, version }) => {
   if (version.indexOf(" ") >= 0) {
     // Semver ranges can have a join of comparator sets
     // e.g. '^3.0.2 || ^4.0.0' or '>=1.2.7 <1.3.0'
-    // Take each version in the range and find the maxSatisfying
-    const rangeSplit = version
-      .split(" ")
-      .map(v => coerce(v))
```

Comment:
it does seem important to still check range validity here? altho then i suppose dist-tags like “next” wouldn’t work.

Reply:
If the range is invalid wouldn't that be a bug in the other package? Do you know how NPM handles invalid ranges here?

Based on the [package.json docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies) on the NPM website, simply taking the last element of the range _should_ work.

Related PR: #4

Thanks for reviewing so fast and sorry for the messy diff — I think my prettier extension settings have changed since I first started writing this package. Force-pushed without the formatting so it's hopefully easier to read.

Reply:
Yeah i think it makes the most sense to just let the package manager handle it.